packages/next/src/server/after/after-context.ts

```diff
@@ -44,23 +46,29 @@ export class AfterContext {
     this.callbackQueue.pause()
   }
 
-  public after(task: AfterTask): void {
+  public after(
+    task: AfterTask,
+    asyncStackTask?: AsyncStackTask | undefined
```

Comment:
looking for better name ideas. `task` is currently taken bc that's the name i used for `after`'s argument and i kinda like that